src/gui/EffectView.cpp

```diff
@@ -45,65 +49,60 @@
 namespace lmms::gui
 {
 
-EffectView::EffectView( Effect * _model, QWidget * _parent ) :
-	PluginView( _model, _parent ),
-	m_bg( embed::getIconPixmap( "effect_plugin" ) ),
```

Comment:
You can go ahead and remove this pixmap from the default/classic theme folders too, if you don't use this elsewhere.